### ISSUE TITLE
[Bug]: Android Navigation buttons lacks content-description for Accessibility

### ISSUE THREAD

**Issue by m-abs** [NativeScript/NativeScript#5948]
### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo]

I've confirmed it myself:
(https://github.com/NativeScript/nativescript/issues)?
![navigation-content-desc-actual-vs-expected](https://user-images.githubusercontent.com/1348705/41367065-45c9f2de-6f3e-11e8-8623-ce1f6bd1bdad.png)

### Tell us about the problem
TalkBack on Android reads Navigation buttons on the ActionBar as just "Button" without any label.

### Which platform(s) does your issue occur on?
Android only.

### Please provide the following version numbers that your issue occurs with:
- CLI: (run `tns --version` to fetch it)
- Cross-platform modules: 4.1.0
- Runtime(s): android 4.1.2
- Plugin(s): none

### Please tell us how to recreate the issue in as much detail as possible.

Demo project link below.
Navigate to any item and notice the "<-" button, this button should be read as "Back button" by TalkBack, but is only read as "Button".

You can see the screenshot from `uiautomationviewer` above, `content-desc` should have the value "Back" but have none.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
https://github.com/m-abs/ns-debugging/tree/master/missing-nav-content-desc

**Note**: I'll submit a PR fixing this
